x-pack/solutions/security/plugins/cloud_security_posture/server/routes/graph/fetch_graph.ts

```diff
@@ -260,34 +266,22 @@ ${
     }
   "}")
 
-// Construct actor and target entity groups
+// Construct actor and target entity groups (using placeholder for non-enriched)
 | EVAL actorEntityGroup = CASE(
     actorEntityType IS NOT NULL AND actorEntitySubType IS NOT NULL,
     CONCAT(actorEntityType, ":", actorEntitySubType),
     actorEntityType IS NOT NULL,
     actorEntityType,
-    actor.entity.id
+    "${NON_ENRICHED_PLACEHOLDER}"
   )
 | EVAL targetEntityGroup = CASE(
     targetEntityType IS NOT NULL AND targetEntitySubType IS NOT NULL,
     CONCAT(targetEntityType, ":", targetEntitySubType),
     targetEntityType IS NOT NULL,
     targetEntityType,
-    target.entity.id
+    "${NON_ENRICHED_PLACEHOLDER}"
   )
 
-| EVAL actorLabel = CASE(actorEntitySubType IS NOT NULL, actorEntitySubType, actor.entity.id)
-| EVAL targetLabel = CASE(targetEntitySubType IS NOT NULL, targetEntitySubType, target.entity.id)
-| EVAL actorEntityType = CASE(
-    actorEntityType IS NOT NULL,
-    actorEntityType,
-    ""
-  )
-| EVAL targetEntityType = CASE(
-    targetEntityType IS NOT NULL,
-    targetEntityType,
-    ""
-  )
 | STATS badge = COUNT(*),
   uniqueEventsCount = COUNT_DISTINCT(CASE(isAlert == false, event.id, null)),
   uniqueAlertsCount = COUNT_DISTINCT(CASE(isAlert == true, event.id, null)),
```

Comment:
```suggestion
 uniqueEventsCount = COUNT_DISTINCT(CASE(isAlert == false, _id, null)),
 uniqueAlertsCount = COUNT_DISTINCT(CASE(isAlert == true, _id, null)),
```

Otherwise the counting is off. It explains previous issue I found: https://github.com/elastic/kibana/pull/238395#issuecomment-3478083756